etc/freertos.mips4k.pic32mx.mk

```diff
@@ -44,8 +44,8 @@ BASEDEFS= -D__PIC32MX__ -D__XC__ -D__XC32 -D__XC -D__FreeRTOS__
 # it contains a full libc system header set that is incompatible with vanilla
 # GCC.
 INCLUDES += -idirafter $(PIC32MXLIBPATH)/pic32mx/include
-BASEDEFS += -DTARGET_PIC32MX -D__32MX795F512H__ \
-	-D__PIC32_FEATURE_SET__=795
+BASEDEFS += -DTARGET_PIC32MX -D__32MX530F128H__ \
+            -D__PIC32_FEATURE_SET__=530
```

Comment:
here the right solution is to use -D__32MXGENERIC__ here, and then in the device specific (plib et al) directory to use the device specific define